common/views/components/WorkEmbed/WorkEmbed.js

```diff
@@ -14,18 +14,15 @@ type Props = {|
 |};
 
 const WorkEmbed = ({ work }: Props) => {
-  const [iiifImageLocation] = work.items.map(item =>
-    item.locations.find(location => location.locationType === 'iiif-image')
-  );
+  const iiifImageLocation = getDigitalLocationOfType(work, 'iiif-image');
   const iiifInfoUrl = iiifImageLocation && iiifImageLocation.url;
-  const iiifImage = iiifImageTemplate(iiifInfoUrl);
-  const imageUrl = iiifImage({ size: '800,' });
-  const imageInfoUrl = convertIiifUriToInfoUri(
-    convertImageUri(imageUrl, 'full')
-  );
+  const iiifImage = iiifInfoUrl && iiifImageTemplate(iiifInfoUrl);
+  const imageUrl = iiifImage && iiifImage({ size: '800,' });
+  const imageInfoUrl =
+    imageUrl && convertIiifUriToInfoUri(convertImageUri(imageUrl, 'full'));
 
-  return (
-    <Fragment>
+  if (imageInfoUrl && iiifImage) {
+    return (
```

Comment:
Ordinarily I think we'd tend to write this as `return imageInfoUrl && iiifImage && (<div>...)`. Then you can get rid of the `else`.